src/openeo_gfmap/fetching/commons.py

```diff
@@ -126,16 +126,18 @@ def _load_collection(
         _load_collection_hybrid, is_stac=is_stac, collection_id_or_url=collection_name
     )
 
+    spatial_is_param = isinstance(spatial_extent, Parameter)
+
     if (
-        temporal_extent is not None
+        temporal_extent is not None and not isinstance(temporal_extent, Parameter)
```

Comment:
for sake of readibility, would it make sense to also introdue temporal_is_param = isinstance(temporal_extent, Parameter)